src/l500/l500-private.cpp

```diff
@@ -197,6 +197,27 @@ namespace librealsense
             _record_action( *this );
         }
 
-
+        // Returns the number of weeks that passed since manufactoring of this device
+        // The Serial Number format is PYWWXXXX:
+        // P � Site Name(ex.�F� for Fabrinet)
+        // Y � Year(ex.�9� for 2019, "0" for 2020, , "1" for 2021  ..etc)
+        // WW � Work Week
+        // XXXX � Sequential number
+        uint8_t weeks_since_factory_calibration(const std::string& serial) {
```

Comment:
I think it should return an `unsigned` -- it's up to the caller to decide how that relates to the struct limitations

Reply:
Formatting -- use the .clang_format file

Reply:
done